fpga/fomu/cpu/ucode.js

```diff
@@ -385,7 +385,7 @@ function disasm(code, words = {}) {
         code &= 0x0FFF;
         suffix = " EXIT";
     }
-    const entry = Object.entries(words).find(function ([_, value]) {
+    const entry = Object.values(words).find(function (value) {
```

Comment:
This is a reverse lookup. We need to find the key corresponding to the given value, so we need the entire _entry_.